lib/stanford-mods/date_parsing.rb

```diff
@@ -85,6 +86,55 @@ def self.facet_string_for_century(date_str)
         end
       end
 
+      # get String sortable value for B.C. if we have  B.C. pattern
+      #  note that these values must *lexically* sort to create a chronological sort.
+      #  We know our data does not contain B.C. dates older than 999, so we can make them
+      #  lexically sort by subtracting 1000.  So we get:
+      #    -700 for 300 B.C., -750 for 250 B.C., -800 for 200 B.C., -801 for 199 B.C.
+      # @param [String] date_str String containing B.C.
+      # @return [String, nil] String sortable -ddd if B.C. in pattern; nil otherwise
+      def self.sortable_year_for_bc(date_str)
```

Comment:
I know this pattern existed before this commit, but something is jumping out at me.

Seeing that this is a class that defines all its methods as class methods (`def self.whatever`), then passes the same variable between all of the methods, seems odd to me.  This either feels like a collection of methods that another class would include or possibly that there should be an initialize method that can take the `date_str` as and argument and then all of these methods could be messages to send to the `DateParsing` class.

```ruby
date_parser = Stanford::Mods::DateParsing.new(my_date_string)

date_parser.sortable_year_for_bc
date_parser.facet_string_for_bc
```

That may be totally out of scope for this PR, but thought I would just give my $0.02 on that.